build/xwalk.gni

```diff
@@ -7,4 +7,13 @@ declare_args() {
   # Whether to build and use Crosswalk's internal extensions (device
   # capabilities, sysapps etc).
   disable_bundled_extensions = false
+
+  # Android: whether the integrity of the Crosswalk runtime library should
+  # be verified when Crosswalk is run in shared mode.
+  verify_xwalk_apk = 0
```

Comment:
This is another case of what I've been saying about following "the spirit of law" (in this case, trying to do things the GN way instead of just copying gyp). In other words, fix the script instead of not using GN the right way :-)

The script accepts ints because gyp uses 1 and 0 instead of true or false, but it should be easy to change the behavior. I can write the patch myself.